api.go

```diff
@@ -311,35 +311,36 @@ func (r *Router) Call(ctx context.Context, bucketID uint64, mode CallMode,
 				// We reproduce here behavior in https://github.com/tarantool/vshard/blob/b6fdbe950a2e4557f05b83bd8b846b126ec3724e/vshard/router/init.lua#L663
 				r.BucketReset(bucketID)
 
-				if destination := vshardError.Destination; destination != "" {
+				if destinationName := vshardError.Destination; destinationName != "" {
```

Comment:
This line has few troubles.
Now we have destination it mean that it might be uuid for old vshard versions or name of replicaset if there is new vshard